### ISSUE TITLE
[CLOSED] Fix bugs

### ISSUE THREAD

**Issue by jeswan** [nyu-mll/jiant-v1-legacy#826]
<a href="https://github.com/W4ngatang"><img src="https://avatars0.githubusercontent.com/u/5520155?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [W4ngatang](https://github.com/W4ngatang)**
_Tuesday Jul 09, 2019 at 03:03 GMT_
_Originally opened as https://github.com/nyu-mll/jiant/pull/826_

----

Fixes some bugs:

- In some task, accidentally wrote labels instead of predictions
- fix bug in write_preds test where we didn't have predictions...
- fix preprocessing bug in broad coverage diagnostics
- use `rte-superglue` classifier for Winogender
- fix bug in Winogender metrics that used logits as predictions
- fix bug in Winogender preprocessing where there were no labels when using BERT
- fix bug in download script where we weren't unzipping downloads
- add v2.0 to all download URLs and update broad coverage diagnostic

Some cleanup:

- remove mention of wmt in evaluation
- update baseline script comment

----

_**[W4ngatang](https://github.com/W4ngatang)** included the following code: https://github.com/nyu-mll/jiant/pull/826/commits_
